packages/thirdweb/src/react/core/hooks/others/useChainQuery.ts

```diff
@@ -139,10 +143,31 @@ function getQueryOptions(chain?: Chain) {
   } as const;
 }
 
+/** @import { defineChain } from "../../../../chains/utils.js" */
 /**
- * @internal
+ * @description Retrieves metadata for a chain such as name, icon, available faucets, block explorers, etc.
+ *
+ * @param chain - Chain to retrieve metadata for, @see {@link defineChain} for how to create a chain from a chain ID.
+ * @returns A React Query result containing the chain metadata, @see {@link ChainMetadata}.
+ *
+ * @example
+ * ```jsx
+ * import { useChainMetadata } from "thirdweb/react";
+ *
+ * const { data: chainMetadata } = useChainMetadata(defineChain(11155111));
+ *
+ * console.log("Name:", chainMetadata.name); // Sepolia
+ * console.log("Faucets:", chainMetadata.faucets); // ["https://thirdweb.com/sepolia/faucet"]
+ * console.log("Explorers:", chainMetadata.explorers); // ["https://sepolia.etherscan.io/"]
+ * ```
+ *
+ * @chain
+ */
+export const useChainMetadata = useChainQuery;
+/**
+ * @deprecated Use `useChainMetadata` instead.
```

Comment:
This hook was not exported, so no need to deprecate this, we can just rename it here